src/js/player.js

```diff
@@ -274,7 +263,54 @@ class Player extends Component {
 
     // Initialize tech instance
     let techComponent = Component.getComponent(techName);
-    this.tech = new techComponent(this, techOptions);
+    this.tech = new techComponent(techOptions);
+
+    this.on(this.tech, 'ready', this.onTechReady);
+    this.on(this.tech, 'usenativecontrols', this.onTechUseNativeControls);
+
+    // firefox doesn't bubble mousemove events to parent. videojs/video-js-swf#37
+    // bugzilla bug: https://bugzilla.mozilla.org/show_bug.cgi?id=836786
+    if (Lib.IS_FIREFOX) {
+      this.on(this.tech, 'mousemove', this.onTechMouseMove);
+    }
+
+    // Listen to every HTML5 events and trigger them back on the player for the plugins
+    this.on(this.tech, 'loadstart', this.onTechLoadStart);
+    this.on(this.tech, 'waiting', this.onTechWaiting);
+    this.on(this.tech, 'canplay', this.onTechCanPlay);
+    this.on(this.tech, 'canplaythrough', this.onTechCanPlayThrough);
+    this.on(this.tech, 'playing', this.onTechPlaying);
+    this.on(this.tech, 'ended', this.onTechEnded);
+    this.on(this.tech, 'seeking', this.onTechSeeking);
+    this.on(this.tech, 'seeked', this.onTechSeeked);
+    this.on(this.tech, 'ended', this.onTechEnded);
+    this.on(this.tech, 'play', this.onTechPlay);
+    this.on(this.tech, 'firstplay', this.onTechFirstPlay);
+    this.on(this.tech, 'pause', this.onTechPause);
+    this.on(this.tech, 'progress', this.onTechProgress);
+    this.on(this.tech, 'durationchange', this.onTechDurationChange);
+    this.on(this.tech, 'fullscreenchange', this.onTechFullscreenChange);
+    this.on(this.tech, 'error', this.onTechError);
+    this.on(this.tech, 'suspend', this.onTechSuspend);
+    this.on(this.tech, 'abort', this.onTechAbort);
+    this.on(this.tech, 'emptied', this.onTechEmptied);
+    this.on(this.tech, 'stalled', this.onTechStalled);
+    this.on(this.tech, 'loadedmetadata', this.onTechLoadedMetaData);
+    this.on(this.tech, 'loadeddata', this.onTechLoadedData);
+    this.on(this.tech, 'timeupdate', this.onTechTimeUpdate);
+    this.on(this.tech, 'ratechange', this.onTechRateChange);
+    this.on(this.tech, 'volumechange', this.onTechVolumeChange);
+    this.on(this.tech, 'texttrackchange', this.onTextTrackChange);
+
+    if (this.controls() && !this.usingNativeControls()) {
+      this.addTechControlsListeners();
+    }
+
+    // Add the tech element in the DOM if it was not already there
+    // Make sure to not insert the original video element if using Html5
+    if (this.tech.el().parentNode !== this.el() && (techName !== 'Html5' || !this.tag)) {
```

Comment:
The parentNode is null for the original video tag when we do the check at this point so we have to make sure to not add it twice. If I remove this check, you end up with two video tag. I don't even know why the parentNode is null but checking `player.tag` seems to do the trick.